src/plugins/coc.js

```diff
@@ -1,28 +1,62 @@
 const { createVerbose } = require('../index')
 
 module.exports = {
+  'cc<=': {
+    priority: -1,
+    fn(a) {
+      const { rand, verbose } = this;
+      const res = rand.next().value % 100 + 1
+      let texts = []
+      let result = ( res <= a )
+      if (res == 1) {
+        result = true
+      }
+      if (res == 100) {
+        result = false
+      }
+      if (res <= a * 0.2) {
+        texts.push('Special!')
+      }
+      verbose.push(createVerbose(
+        'dice',
+        `ccb<=${a}`,
+        res,
+        texts.join(',')
+      ))
+      return result
+    }
+  },
   'ccb<=': {
     priority: -1,
     fn(a) {
       const { rand, verbose } = this;
       const res = rand.next().value % 100 + 1
       let texts = []
-      if (res <= a * 0.1) {
+      let result = ( res <= a )
+      if (res == 1) {
+        result = true
+      }
+      if (res == 100) {
+        result = false
```

Comment:
:eyes:
[IMHO]
Probably unnecessary because 5% rule. (lines 47, 51)
It for the future that into shared functional within consider 1 or 100` on cc/bcc.

e.g. `const res = primary_roll_result(res, a)`

This pull request do not refactoring as much as possible.

Reply:
oh.. thank you. I made a mistake.